docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,443 @@
+  on-chain configuration settings, such as the consensus type and allowed
+  transaction types, that will be available to the new validator nodes once they
+  join the network.
+
+.. note::
+
+   The first validator node on the network has no special meaning, other than
+   being the node that created the genesis block. Sawtooth has no concept of a
+   "head node" or "master node". Once multiple nodes are up and running, each
+   node has the same genesis block and treats all other nodes as peers.
+
+
+Docker: Start a Multiple-node Sawtooth Network
+==============================================
+
+#. If the first validator node is running, stop the Sawtooth components and run
+   ``docker-compose down``, as described in :ref:`stop-sawtooth-docker-label`.
+
+#. Create a Sawtooth application development environment as described in
+   :doc:`docker`, with the following important differences:
+
+   a. Download the Docker Compose file for a multiple-node network,
+      `sawtooth-default-poet.yaml <./sawtooth-default-poet.yaml>`_. Save this
+      file in the same directory as the single-node compose file
+      (``sawtooth-default.yaml``).
+
+   #. Use the following command to start the multiple-node Sawtooth network:
+
+      .. code-block:: console
+
+         user@host$ docker-compose -f sawtooth-default-poet.yaml up
+
+   #. This Compose file creates five validator nodes, numbered from 0 to 4.
+      Note the container names for the Sawtooth components on each node:
+
+      ``validator-0``:
+
+        * ``sawtooth-poet-validator-0``
+        * ``sawtooth-poet-rest-api-0``
+        * ``sawtooth-poet-settings-tp-0``
+        * ``sawtooth-poet-intkey-tp-python-0``
+        * ``sawtooth-poet-xo-tp-python-0``
+        * ``sawtooth-poet-validator-registry-tp-0``
+
+      ``validator-1``:
+
+        * ``sawtooth-validator-default-1``
+        * ``sawtooth-poet-rest-api-1``
+        * ``sawtooth-settings-tp-default-1``
+        * ``sawtooth-poet-intkey-tp-python-1``
+        * ``sawtooth-poet-xo-tp-python-1``
+        * ``sawtooth-poet-validator-registry-tp-1``
+
+      ... and so on.
+
+   #. Note that there is only one shell container:
+
+      * ``sawtooth-poet-shell``
+
+#. Configure the transaction family settings on the first validator node.
+
+   a. Connect to the ``sawtooth-poet-validator-0`` container.
+
+      .. code-block:: console
+
+         user@host$ docker exec -it ``sawtooth-poet-validator-0`` bash
+         root@validator-0#
+
+   #. Run the following command to create and submit a batch containing the
+      new settings:
+
+      .. code-block:: console
+
+         root@validator-0# sawset proposal create --url http://rest-api:8008 \
+         --key /root/.sawtooth/keys/my_key.priv \
+         sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0"},{"family":"sawtooth_settings", "version":"1.0"},{"family":"xo", "version":"1.0"},{"family":"sawtooth_validator_registry", "version":"1.0"}]'
```

Comment:
Sorry I missed this on the first go around. Do others feel this step is necessary? I think the validator will accept all txn families unless given an explicit list. Since these instructions are just for adding a second node I'd prefer to pare back anything that is optional.

Reply:
The issue is that the "first" procedure restricts the set of txn families, so this step is necessary to add xo.  I think that the original proc had this step to show that restriction is possible. If that's not an important thing, I could remove the "sawset proposal create" step everywhere, which would simplify all the procedures. 

I agree that simplicity is better.  Opinons from others?

Reply:
Yeah, I would remove the sawset proposal for restricting transaction_families since we want to focus only on those steps that are necessary to add another node.

Reply:
I'm okay with simplifying it (not including restricting transaction families at all). We would want some page to document that feature though so we can refer to it. (Maybe we have that already in another location?)

Reply:
@vaporos - I can move the info about restricting txn families to the Transaction Families Specification. (I can find a spot in the intro text to add this information).  I suggest changing the example to show Settings, BlockInfo, and Identity (not intkey & xo), because that's a more useful/realistic example.